project-base/docker/conf/docker-compose-mac.yml.dist

```diff
@@ -97,15 +98,8 @@ services:
             - discovery.type=single-node
 
     microservice-product-search:
-        build:
-            context: ../microservices/product-search/docker
-            args:
-                www_data_uid: 501
-                www_data_gid: 20
+        image: shopsys/shopsys:microservice-test
```

Comment:
microservice test?

Reply:
Well until this CR is done I want to make clear that this is just test right now.